### PR TITLE
docs: add Triton Inference Server documentation

### PR DESCRIPTION
**Pull request by mattdibi**
Added Triton Inference Server documentation featuring the newly added AI Model Encryption support.

**Related Issue:** Wait for #3986 to be merged before closing this PR.

### CODE REVIEW

File: _posts/builtin/2017-10-23-nvidiatm-triton-server-inference-engine.md [hunk 1]
```diff
@@ -0,0 +1,170 @@
+---
+layout: page
+title:  "Nvidia™ Triton Server inference engine"
+categories: [builtin]
+---
+
+The [Nvidia™ Triton Server](https://developer.nvidia.com/nvidia-triton-inference-server) is an open-source inference service software that enables the user to deploy trained AI models from any framework on GPU or CPU infrastructure. It supports all major frameworks like TensorFlow, TensorRT, PyTorch, ONNX Runtime, and even custom framework backend. With specific backends, it is also possible to run Python scripts, mainly for pre-and post-processing purposes, and exploit the [DALI](https://github.com/triton-inference-server/dali_backend) building block for optimized operations. For more detail about the Triton Server, please refer to the official [website](https://github.com/triton-inference-server/server).
+
+The ESF Triton Server component is the implementation of the inference engine APIs and provides methods for interacting with a local or remote Nvidia™ Triton Server. As presented below, the component enables the user to configure a local server running on the gateway or to communicate to an external server to load specific models.
```
Comment (nicolatimeus) [inline _posts/builtin/2017-10-23-nvidiatm-triton-server-inference-engine.md]: `s/ESF/Kura/g`

File: _posts/builtin/2017-10-23-nvidiatm-triton-server-inference-engine.md [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+zip -vr tf_autoencoder_fp32.zip tf_autoencoder_fp32/
+```
+
+then encrypt it with the AES 256 algorithm using the following `gpg` command:
+
+```bash
+gpg --armor --symmetric --cipher-algo AES256 tf_autoencoder_fp32.zip
+```
+
+The resulting archive `tf_autoencoder_fp32.zip.asc` can be transferred to the target machine and will be decrypted by Kura.
```
Comment (nicolatimeus) [inline _posts/builtin/2017-10-23-nvidiatm-triton-server-inference-engine.md]: Maybe we can mention explicitly that the encrypted model must be placed in the directory configured as **Local model repository path**.

Comment (nicolatimeus) [inline _posts/builtin/2017-10-23-nvidiatm-triton-server-inference-engine.md]: Maybe we can mention explicitly that the encrypted model must be placed in the directory configured as **Local model repository path**.

Comment (mattdibi) [reply _posts/builtin/2017-10-23-nvidiatm-triton-server-inference-engine.md]: Should be ok now.

Comment (mattdibi) [reply _posts/builtin/2017-10-23-nvidiatm-triton-server-inference-engine.md]: Should be ok now.

**Review by nicolatimeus — APPROVED** [review]

**Pull request merged**